BlueprintUI/Sources/Layout/Transform.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+
+/// Changes the transform of the wrapped element.
+public struct Transform: Element {
```

Comment:
I'm going to bikeshed just a bit and suggest `Transformed`, to match the style of Aligned, ConstrainedSize, etc.

Reply:
+1